test/DynamicFees.t.sol

```diff
@@ -168,22 +150,26 @@ contract TestDynamicFees is Test, Deployers, GasSnapshot {
     }
 
     function testUpdateRevertsOnStaticFeePool() public {
-        (PoolKey memory staticPoolKey,) = Deployers.createAndInitPool(manager, IHooks(address(0)), 3000, SQRT_RATIO_1_1);
+        (key,) = initPool(currency0, currency1, IHooks(address(0)), 3000, SQRT_RATIO_1_1, ZERO_BYTES);
         vm.expectRevert(IFees.FeeNotDynamic.selector);
-        manager.updateDynamicSwapFee(staticPoolKey);
+        manager.updateDynamicSwapFee(key);
     }
 
     function testDynamicFeesCacheNoOtherHooks() public {
+        (key,) = initPoolAndAddLiquidity(
+            currency0, currency1, dynamicFeesNoHook, FeeLibrary.DYNAMIC_FEE_FLAG, SQRT_RATIO_1_1, ZERO_BYTES
+        );
+
         dynamicFeesNoHook.setFee(123);
-        manager.updateDynamicSwapFee(key2);
+        manager.updateDynamicSwapFee(key);
 
         IPoolManager.SwapParams memory params =
             IPoolManager.SwapParams({zeroForOne: true, amountSpecified: 100, sqrtPriceLimitX96: SQRT_RATIO_1_2});
         PoolSwapTest.TestSettings memory testSettings =
             PoolSwapTest.TestSettings({withdrawTokens: true, settleUsingTransfer: true});
 
         vm.expectEmit(true, true, true, true, address(manager));
-        emit Swap(key.toId(), address(swapRouter), 100, -99, 79228162514264329670727698910, 1e18, -1, 0);
+        emit Swap(key.toId(), address(swapRouter), 100, -98, 79228162514264329749955861424, 1e18, -1, 123);
```

Comment:
is the change here from using a new poolKey with diff settings?

Reply:
Yes its because this was a fees test that wasnt actually testing with a fee lol